sdk/pom.xml

```diff
@@ -39,6 +39,11 @@
       <artifactId>jackson-databind</artifactId>
       <version>2.15.1</version>
     </dependency>
```

Comment:
Could you please remove this dependency. We don't want to enforce anything beyond Jackson Core and Jackson Databind.

Dapr Java SDK users can decide to use whatever Jackson module they need, we just need to have a way to allow them to customize the serializer.